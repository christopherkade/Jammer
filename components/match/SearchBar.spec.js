import { shallowMount } from '@vue/test-utils'
import cmp from './SearchBar.vue'

describe('[UNIT] Testing the /match Search Bar component', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(cmp, {
      data() {
        return {
          emailInput: ''
        }
      }
    })
  })

  it('Snapshot testing', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })

  describe('Component validation', () => {
    it('should render the main (parent) component', () => {
      expect(wrapper.is(cmp)).toBe(true)
    })
  })
})
