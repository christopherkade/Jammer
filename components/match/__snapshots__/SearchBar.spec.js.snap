// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[UNIT] Testing the /match Search Bar component Snapshot testing 1`] = `
"<div class=\\"match-search\\"><div class=\\"field has-addons\\"><div class=\\"control search-control\\"><input type=\\"email\\" placeholder=\\"Musician's Email\\" class=\\"input input-searchbar\\"></div> <div class=\\"control\\"><a class=\\"button search-button\\">
        +
      </a></div></div></div>"
`;
